#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("sigma_s", double_t, 0, "The standard deviation of the Gaussian", 15.0, 0.0, 100.0)
gen.add("sigma_r", double_t, 0, "The standard deviation of the Gaussian used to control how much an adjacent pixel is downweighted because of the intensity difference (depth in our case)", 0.05, 0.0, 1.0)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "BilateralFilter"))
